Update readme (#64)
Huemul/subilo: README.md

[![Rust](https://github.com/huemul/subilo/workflows/Rust/badge.svg)](https://github.com/Huemul/subilo/actions?query=workflow%3ARust)

Subilo is a tool to setup continuous deployments for applications running on

> machines with no external integrations like IoT devices and VPSs.

Reviewer: ```suggestion
machines with no external integrations, like IoT devices and VPSs.
```

#### How it works: 
Subilo is a small server that lives on your app's machine and listens for
authenticated HTTP webhooks. These webhooks have information about what app
should be deployed matching the Subilo configuration file (`.subilorc`).
This configuration file also defines what steps should be taken to successfully

> deploy an application, for example: `git pull`, `./restart-server` and `./notify`.

Reviewer: ```suggestion
deploy an application, for example: `git pull`, pull the latest Docker image and
restart the server.
```

Webhook:

> This webhook is usually sent from a CI after the tests passed.

Reviewer: ```suggestion
This webhook is usually sent from a CI run after the tests passed.
```

### Configuration

Create a `.subilorc` file with the required configuration.

> A `.subilorc` example can be found [here](https://github.com/huemul/subilo/blob/master/configuration.md).

Reviewer: ```suggestion
A complete example can be found [here](https://github.com/huemul/subilo/blob/master/configuration.md).
```
